### DISCUSSION TITLE
Wallets showing up multiple times

### DISCUSSION THREAD

**Discussion by SumiBisht** [WalletConnect/walletconnect-monorepo#3400] [Developer Support]
**Describe the bug**
I am seeing some wallets multiple times in modal
**SDK Version (if relevant)** 
"@walletconnect/ethereum-provider": "^2.8.6",
"@walletconnect/modal": "^2.5.9"

**To Reproduce**
Just Open the modal I cant paste the Video here
My code:
`async function InitializeProviderClient() {
    // console.log('Initializing client');
    if(providerClient === undefined){ 
      const ethereumProvider = await EthereumProvider.init({
        projectId,
        showQrModal: true,
        qrModalOptions: { themeMode: 'light' },
        chains: [1,56],
        methods: ['eth_sendTransaction', 'personal_sign'],
        events: ['chainChanged', 'accountsChanged'],
        metadata: {
          name: 'app.expand.network',
          description: 'GUI for expandAPIs',
          url: 'https://app.expand.network',
          icons: [''],
        },
      });
      console.log('InitP');
      setProviderClient(ethereumProvider);
  }
  }`
` const connectToWalletV2 = async () => {
    try {
      setLoading(true);
      // checkIfExtensionIsAvailable();
      if (providerClient) {
        // console.log("connecting to v2");
        await providerClient.connect();
        console.log('connectTowalletV2')
        //  console.log('connected: ',providerClient);
        const library = new ethers.providers.Web3Provider(providerClient);
        await subscribeProvider(providerClient);
        //  console.log('subscribed');
        setLibrary(library);
        setWalletAddress(library);
        setLoading(false);
        setProvider(providerClient);
      }
    } catch (error) {
      setLoading(false);
      console.log(
        error,
        ' Client not set got this error on connectToWallet catch block while connecting the wallet'
      );
    }
  };`

**Expected behavior**
It should show supported wallets 
**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser : Chrome

**Smartphone (please complete the following information):**

**Additional context**

**Comment by glitch-txs** [comment]
hi do you have a public repo/example where we can reproduce this?
